test/upgrade/installation/serverless.go

```diff
@@ -53,3 +53,64 @@ func UpgradeServerless(ctx *test.Context) error {
 
 	return nil
 }
+
+func DowngradeServerless(ctx *test.Context) error {
+	const subscription = "serverless-operator"
+
+	if err := test.DeleteSubscription(ctx, subscription, test.OperatorsNamespace); err != nil {
+		return err
+	}
+
+	if err := test.DeleteClusterServiceVersion(ctx, test.Flags.CSV, test.OperatorsNamespace); err != nil {
```

Comment:
So the idea is to keep the CRDs, install previous and test again. In the future we plan to use new crd versions and even if we have a conversion webhook when we move to a previous version I suspect the CR will not get downgraded.